test/pkg/integration/integration.go

```diff
@@ -206,32 +206,30 @@ type RpcClient struct {
 
 func (r *RpcClient) Call(serviceMethod string, args any, reply any) error {
 	var err error
-	var new *RpcClient
+	cl := r
```

Comment:
There are a couple spots (below) where we can return w/o closing the RpcClient. Should we add a `defer cl.Close()`?